railties/lib/rails/generators/rails/app/templates/test/test_helper.rb.tt

```diff
@@ -4,7 +4,7 @@ require "rails/test_help"
 
 class ActiveSupport::TestCase
   # Run tests in parallel with specified workers
-<% if defined?(JRUBY_VERSION) || Gem.win_platform? -%>
+<% if !Process.respond_to?(:fork) -%>
   parallelize(workers: :number_of_processors, with: :threads)
 <%- else -%>
   parallelize(workers: :number_of_processors)
```

Comment:
I'd flip this conditional:

```suggestion
<% if Process.respond_to?(:fork) -%>
  parallelize(workers: :number_of_processors)
<%- else -%>
  parallelize(workers: :number_of_processors, with: :threads)
```

Reply:
@eugeneius I've flipped this conditional.